gpustack/worker/rpc_server.py

```diff
@@ -38,25 +43,42 @@ def start(
         vendor: str,
         log_file_path: str,
         cache_dir: str,
+        bin_dir: str,
         args: Optional[List[str]] = None,
     ):
         setproctitle.setproctitle(f"gpustack_rpc_server_process: gpu_{gpu_index}")
         add_signal_handlers()
 
         with open(log_file_path, "w", buffering=1, encoding="utf-8") as log_file:
             with redirect_stdout(log_file), redirect_stderr(log_file):
-                RPCServer._start(port, gpu_index, vendor, cache_dir, args)
+                RPCServer._start(port, gpu_index, vendor, cache_dir, bin_dir, args)
 
     def _start(
         port: int,
         gpu_index: int,
         vendor: str,
         cache_dir: str,
+        bin_dir: str,
```

Comment:
if we allow `bin_dir` to be None, should we declare to `bin_dir: Optional[str] = None`?